connect-widget/app/src/pages/connect/components/connect-button.tsx

```diff
@@ -25,15 +25,15 @@ const ConnectButton = ({
     sm: 125,
   });
 
-  const garmin = useBreakpointValue({
-    base: <Text p="4">+</Text>,
-    sm: <Text p="4">Coming Soon!</Text>,
-  });
+  // const garmin = useBreakpointValue({
+  //   base: <Text p="4">+</Text>,
+  //   sm: <Text p="4">Coming Soon!</Text>,
+  // });
 
   return (
     <Box>
       <Center>
-        {provider.name === "garmin" ? (
+        {/* {provider.name === "garmin" ? (
```

Comment:
I assume the comments are here on purpose so we can reference this in the future?

Reply:
This was commented out in another PR where we temporarily disabled Garmin, not sure why it's showing as difference... but yeah, we'll reenable this soon.